packages/react-select/src/Select.tsx

```diff
@@ -628,13 +630,28 @@ export default class Select<
     this.instancePrefix =
       'react-select-' + (this.props.instanceId || ++instanceId);
     this.state.selectValue = cleanValue(props.value);
+
+    // If `value` or `defaultValue` props are not empty then announce them
+    // when the Select is focused
+    if (this.state.selectValue.length) {
+      this.state.ariaSelection = {
+        value: this.state.selectValue as OnChangeValue<Option, IsMulti>,
+        options: this.state.selectValue,
+        action: 'initial-input-focus',
+        name: this.props.name,
+      };
+    }
   }
+
   static getDerivedStateFromProps(
     props: Props<OptionBase, boolean, GroupBase<OptionBase>>,
     state: State<OptionBase, boolean, GroupBase<OptionBase>>
   ) {
-    const { prevProps, clearFocusValueOnUpdate, inputIsHiddenAfterUpdate } =
-      state;
+    const {
+      prevProps,
+      clearFocusValueOnUpdate,
+      inputIsHiddenAfterUpdate,
+    } = state;
```

Comment:
Different version of Prettier?

Reply:
Haha, I yeh. I thought I removed the re-formatting changes before pushing.